### ISSUE TITLE
Sharing full external drive with slash "/" shows system folders instead of external drive

### ISSUE THREAD

**Issue by oncre** [openmediavault/openmediavault#413]
### Description of issue/question

 Armbian is running on the onboard eMMC and I have an external USB drive that I want to share using SMB. When I click "add share" I get a window where I have to put the name, select the volume (mounted through OMV) and the folder to share relative to the volume that contains it. I read that if I want to share the whole USB disk and not a folder inside it I have to put a slash "/" in the path field, which I did. Then when I connected to the server instead of showing the shared usb drive it showed the root of system volume (eMMC) and not the USB drive.

### Steps to reproduce issue
(Include debug logs if possible and relevant.)

### Versions report
OMV Version 5
Armbian 5.90.190705 Debian buster dev_5.1.15  for Orange Pi 3 

*UPDATED FOR CLARIFICATIONS

**Comment by votdev** [comment]
Can you please describe the problem more detailed? I don't know what you want.

**Comment by oncre** [comment]
Sure, sorry fo my ambiguity. Armbian is running on the onboard eMMC and I have an external USB drive that I want to share using SMB. When I click "add share" I get a window where I have to put the name, select the volume (mounted through OMV) and the folder to share relative to the volume that contains it. I read that if I want to share the whole USB disk and not a folder inside it I have to put a slash "/" in the path field, which I did. Then when I connected to the server instead of showing the shared usb drive it showed the root of system volume (eMMC) and not the USB drive.
